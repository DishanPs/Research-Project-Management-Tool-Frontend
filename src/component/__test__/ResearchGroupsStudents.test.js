const ResearchGroupsStudents = require('./ResearchGroupsStudentsTest');

test("User fetched Link name should be in list", async () => {
  expect.assertions(1);
  const data = await ResearchGroupsStudents.getGroups();
  expect("Kamal").toEqual("Kamal");
});
