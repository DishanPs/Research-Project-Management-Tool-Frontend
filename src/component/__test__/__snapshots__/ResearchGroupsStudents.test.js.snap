// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResearchGroupsStudents Change the component when clicked 1`] = `
<a
  className="normal"
  href="https://research-project-management.netlify.app/creategroups"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  ResearchGroupsStudents
</a>
`;

exports[`ResearchGroupsStudents Change the component when clicked 2`] = `
<a
  className="hovered"
  href="https://research-project-management.netlify.app/creategroups"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  ResearchGroupsStudents
</a>
`;

exports[`ResearchGroupsStudents Change the component when clicked 3`] = `
<a
  className="normal"
  href="https://research-project-management.netlify.app/creategroups"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  ResearchGroupsStudents
</a>
`;
